components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtValidator.java

```diff
@@ -77,6 +77,8 @@
             "for tenantDomain:%s.";
     private static final String CLAIM_NOT_AVAILABLE = "Local claim %s is not available in the server " +
             "for tenantDomain:%s.";
+    private static final String SP_CLAIM_NOT_AVAILABLE = "Application Claim URI '%s' is not defined " +
+            "for application:%s.";
```

Comment:
shall we add a space after : mark

Reply:
I didn't add a space there to keep consistency with other error logs.